extensions/arc/notebooks/arcDeployment/deploy.postgres.existing.arc.ipynb

```diff
@@ -161,13 +163,18 @@
                 "\n",
                 "workers_option = f' -w {server_group_workers}' if server_group_workers else \"\"\n",
                 "port_option = f' --port \"{server_group_port}\"' if server_group_port else \"\"\n",
+				"engine_version_option = f' --engine-version -ev {server_group_engine_version}' if server_group_engine_version else \"\"\n",
+				"volume_size_backups_option = f' --volume-size-backups -vsb {server_group_volume_size_backups}' if server_group_volume_size_backups else \"\"\n",
+				"volume_size_data_option = f' volume-size-data -vsd {server_group_volume_size_data}' if server_group_volume_size_data else \"\"\n",
+				"volume_size_logs_option = f' --volume-size-logs -vsl {server_group_volume_size_logs}' if server_group_volume_size_logs else \"\"\n",
+                "extensions_option = f' --extensions \"{server_group_extensions}\"' if server_group_extensions else \"\"\n",
                 "cores_request_option = f' -cr \"{server_group_cores_request}\"' if server_group_cores_request else \"\"\n",
                 "cores_limit_option = f' -cl \"{server_group_cores_limit}\"' if server_group_cores_limit else \"\"\n",
                 "memory_request_option = f' -mr \"{server_group_memory_request}Mi\"' if server_group_memory_request else \"\"\n",
                 "memory_limit_option = f' -ml \"{server_group_memory_limit}Mi\"' if server_group_memory_limit else \"\"\n",
                 "\n",
                 "os.environ[\"AZDATA_PASSWORD\"] = postgres_password\n",
-                "cmd = f'azdata arc postgres server create -n {server_group_name} -scd {postgres_storage_class_data} -scl {postgres_storage_class_logs} -scb {postgres_storage_class_backups}{workers_option}{port_option}{cores_request_option}{cores_limit_option}{memory_request_option}{memory_limit_option}'\n",
+				"cmd = f'azdata arc postgres server create -n {server_group_name} -scd {postgres_storage_class_data} -scl {postgres_storage_class_logs} -scb {postgres_storage_class_backups} -w {workers_option} --port {port_option} -ev {engine_version_option} -vsb {volume_size_backups_option} -vsd {volume_size_data_option} -vsl {volume_size_logs_option} --extensions {extensions_option} -cr {cores_request_option} -cl {cores_limit_option} -mr {memory_request_option} -ml {memory_limit_option}'\n",
```

Comment:
You don't have to add arguments to these options.  The workers_option variable for example already includes the -w so this would duplicate it.